pkg/yang/types_builtin.go

```diff
@@ -568,12 +568,12 @@ func pow10(e uint8) uint64 {
 // A EnumType represents a mapping of strings to integers.  It is used both
 // for enumerations as well as bitfields.
 type EnumType struct {
-	last     int64 // maximum value assigned thus far
-	min      int64 // minimum value allowed
-	max      int64 // maximum value allowed
-	unique   bool  // numeric values must be unique (enums)
-	toString map[int64]string
-	toInt    map[string]int64
+	last     int64            // maximum value assigned thus far
+	min      int64            // minimum value allowed
+	max      int64            // maximum value allowed
+	unique   bool             // numeric values must be unique (enums)
+	ToString map[int64]string `json:",omitempty"`
+	ToInt    map[string]int64
```

Comment:
Could you add comments for these now that they're public?

Reply:
added - I added `name` and `value` from the [RFC 6020 9.6.4](https://datatracker.ietf.org/doc/html/rfc6020#page-120:~:text=cannot%20be%20restricted.-,9.6.4,-.%20%20The%20enum%20Statement)